packages/debugger/src/panels/variables/tree.tsx

```diff
@@ -239,9 +240,11 @@ const VariableComponent = (props: IVariableComponentProps): JSX.Element => {
   return (
     <li
       onClick={(e): Promise<void> => onVariableClicked(e)}
-      onMouseDown={() => {
+      onMouseDown={e => {
+        e.stopPropagation();
         onSelection(variable);
       }}
+      className={value ? '' : 'jp-DebuggerVariables-noCopy'}
```

Comment:
Maybe more genetic name for this class, e.g. `jp-DebuggerVariables-emptyValue` would be better?

Reply:
I agree. It can also be useful for the rich variable inspector.